### PR TITLE
Updated .github/PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 | Q             | A
 | ------------- | ---
-| Branch?       | "master" for new features / 2.7, 2.8 or 3.1 for fixes
```
Comment (lyrixx) [inline .github/PULL_REQUEST_TEMPLATE.md]: 1/ It's boring to remove surrounding quotes  ;)
2/ 3.2 is now a "bug fix" branch

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -9,3 +9,6 @@
 | Fixed tickets | comma-separated list of tickets fixed by the PR, if any
 | License       | MIT
 | Doc PR        | reference to the documentation PR, if any
+
+---
```
Comment (lyrixx) [inline .github/PULL_REQUEST_TEMPLATE.md]: It's cleaner to had an `<hr />` IHMO

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (javiereguiluz) [inline .github/PULL_REQUEST_TEMPLATE.md]: Problem is that "master" is a common word in English. Removing the quotes may confuse the less experienced developers (although I'm divided myself about this).

**Comment by nicolas-grekas** [comment]
see #20381 also

**Comment by lyrixx** [comment]
Closing in favor or #20381

**Pull request closed without merging**